app/code/Transoft/Blog/view/frontend/templates/blog_post_list.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NotFoundException;
+use Transoft\Blog\Block\Bloglist;
+?>
+<?php /** @var Bloglist $block*/
+?>
+<?php
+try {
+    $jsonData = $block->getConfig();
+} catch (LocalizedException $e) {
+    $block->_logger->warning($e);
+}
+?>
+
+<?php try {
+    if ($block->isTypesMatching()): ?>
+        <div data-mage-init='{"blogList": <?= $block->escapeHtml($jsonData) ?> }' class="blog-list">
+        </div>
+    <?php endif;
+} catch (NotFoundException $e) {
+    $block->_logger->warning($e);
+} ?>
```

Comment:
Please remove try/catch in block template

Reply:
fixed